sdk/python/tests/test_sanity_errors.py

```diff
@@ -0,0 +1,150 @@
+
+    def test_int32_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.number32 = []
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_int64_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.number64 = 9223372036854775808
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_uint8_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.u_number8 = -1
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_uint16_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.u_number16 = 'not an uint'
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_uint16_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.u_number16 = 'not an uint'
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_uint32_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.u_number32 = 4294967296
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_uint64_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.u_number64 = 18446744073709551616
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYError,
+            self.crud.create, self.ncc, runner)
+
+    # but invalid input is sent to the device
+    def test_string_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.name = ['name_str']
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+        # self.crud.create(self.ncc, runner)
+
+    def test_empty_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.emptee = '0'
+        
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_boolean_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.bool_value = ''
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+    def test_enum_invalid(self):
+        runner = self._create_runner()
+        runner.ytypes.built_in_t.enum_value = 'not an enum'
+        # self.crud.create(self.ncc, runner)
+        self.assertRaises(YPYModelError,
+            self.crud.create, self.ncc, runner)
+
+
```

Comment:
can you also add a few test cases here for testing the errors dictionary when YPYModelError is raised?